data/json/items/ammo.json

```diff
@@ -976,19 +977,21 @@
     "type": "AMMO",
     "id": "charcoal",
     "category": "spare_parts",
-    "price": 200,
-    "price_postapoc": 50,
+    "price": 250,
+    "price_postapoc": 80,
```

Comment:
It *should* be going down because count increased giving a lower per-unit price, but it's been a little while so I'm going off memory here.

Reply:
Ah, right, though I think it's stack size, not count. So price 20c->10c and barter value 5c->(80/25)... might want to check that it doesn't cause issues due to rounding, since that doesn't divide evenly.

Another thing: since you set it up so that coal and charcoal have equivalent energy per unit, then they should be of the same value (price) and this is not currently the case since coal has price (200 / stack_size 200) and barter (100 / stack_size 200)